src/web/components/ClientSideTokenGeneration/CstgDomainsTable.scss

```diff
@@ -43,7 +47,34 @@
 
     &-right {
       display: flex;
-      align-items: center;
+      justify-content: right;
+    }
+
+    .domains-search-bar-container {
+      display: flex;
```

Comment:
it might be worth adding `align-items: center` here so that the search icon lines up with the text better: 

Before
![image](https://github.com/IABTechLab/uid2-self-serve-portal/assets/137864392/381a52a3-7c48-4529-871f-1a3e20608986)


After
![image](https://github.com/IABTechLab/uid2-self-serve-portal/assets/137864392/57be09f6-77a9-4eb3-a0ff-39a0051a1c89)